### DISCUSSION TITLE
Why can't I add inline js to DOM using el('div', {...})?

### DISCUSSION THREAD

**Discussion by PatAbah** [redom/redom#449] [Q&A] [answered]
When I do this:
`node = el('div.indiv-comment cbubble', { id: "send_reply('reply_someone')"}) 
           `
I get `<div class="indiv-comment" id="send_reply('reply_someone')"></div>` as expected.

However, when I replace the 'id' attribute with ondblclick as in:
`node = el('div.indiv-comment cbubble', { ondblclick: "send_reply('reply_someone')"}) 
           `
Nothing appears... It results in: `<div class="indiv-comment"></div>`

**Comment by pakastin** [comment, marked as answer]
ondblclick is property, which means you can use function there:

```{ ondblclick: e => send_reply('reply_someone') }```

**Reply by PatAbah** [reply]
Thanks! This fixed my code.

**Reply by pakastin** [reply]
Perfect! 👌
